Makefile

```diff
@@ -120,13 +120,13 @@ uninstall:
 coverage: ## generate coverprofiles from the unit tests
 	@echo "🐳 $@"
 	@( for pkg in $(filter-out ${INTEGRATION_PACKAGE},${PACKAGES}); do \
-		go test -i -race -tags "${DOCKER_BUILDTAGS}" -test.short -coverprofile="../../../$$pkg/coverage.txt" -covermode=atomic $$pkg || exit; \
-		go test -race -tags "${DOCKER_BUILDTAGS}" -test.short -coverprofile="../../../$$pkg/coverage.txt" -covermode=atomic $$pkg || exit; \
+		go test -i -race -tags "${DOCKER_BUILDTAGS}" -test.short -coverprofile=coverage.txt -covermode=atomic $$pkg || exit; \
+		go test -race -tags "${DOCKER_BUILDTAGS}" -test.short -coverprofile=coverage.txt -covermode=atomic $$pkg || exit; \
 	done )
 
 coverage-integration: ## generate coverprofiles from the integration tests
 	@echo "🐳 $@"
-	go test -race -tags "${DOCKER_BUILDTAGS}" -test.short -coverprofile="../../../${INTEGRATION_PACKAGE}/coverage.txt" -covermode=atomic ${INTEGRATION_PACKAGE}
+	go test -race -tags "${DOCKER_BUILDTAGS}" -test.short -coverprofile=coverage.txt -covermode=atomic ${INTEGRATION_PACKAGE}
```

Comment:
I'm not sure this is useful, because it is just a coverage of `integration` package, not the aggregated coverage of the entire project.

Also, `integration` package seems not available yet.